### DISCUSSION TITLE
How to get the column count?

### DISCUSSION THREAD

**Discussion by rzmk** [dathere/qsv#1993] [Q&A] [answered]
Wondering what way to get the column count for a CSV (potentially the fastest way too) with qsv.

**Comment by jqnatividad** [comment]
`qsv headers` is the fastest way to do it.

However, you need to parse it out.

Thinking about it, perhaps we can add a `--json` option to it, or if you just really just want the column count, a `--just-count` option?

**Reply by rzmk** [reply]
I think having a way to get the column count similar to `qsv count` can work well so `--just-count` makes sense for my use case.

**Comment by rzmk** [comment, marked as answer]
Thanks resolved in https://github.com/jqnatividad/qsv/issues/2004.
